composer.json

```diff
@@ -41,14 +41,14 @@
         "phing/phing": "^2.17.4",
         "phpstan/phpstan": "^1.10.46",
         "phpunit/phpunit": "^10.4.2",
-        "roave/infection-static-analysis-plugin": "^1.33",
+        "roave/infection-static-analysis-plugin": "^1.34",
         "vimeo/psalm": "^5.16.0"
     },
     "config": {
         "optimize-autoloader": true,
         "sort-packages": true,
         "platform": {
-            "php": "8.1.99"
+            "php": "8.2.99"
```

Comment:
Wait, this change is not valid: we're still supporting `8.1.x`, so we should pin the `composer update` operations performed (by renovate and similar) to PHP 8.1, unless we drop support for it.

Reply:
Should I add some Ci to check we can install symfony 7 ?
Because I think symfony 7 require php 8.2.

Reply:
I'm also fine with dropping PHP 8.1 and calling it a day, tbh, but don't have a clean solution right now :thinking:

Reply:
8.1 is still used by around 37% of requests to packagist.org, 8.2 and up is only 33%. And the installation instructions state: Just download the latest PHAR and run it. Just sayin...

Reply:
The percentage of existing users on 8.1 is totally irrelevant: we have old releases for that, which are very much valid.

Problem here is testing Symfony 7 while still enforcing a `8.2.99` platform: one possible way is `--ignore-platform-req=php` for CI pipelines targeting `latest` dependencies, but it introduces instability.

At that point, I'd say that dropping PHP 8.1 is the simplest way forward instead: I got shit to do, and making everyone happy won't get me paid anyway :P